tests/security/pam/pam_config.pm

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: PAM tests for pam-config, create, add or delete service
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#70345, tc#1767580
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use registration qw(add_suseconnect_product remove_suseconnect_product);
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Create a simple Unix authentication configuration, all backup files will not be deleted
+    assert_script_run 'pam-config --create';
```

Comment:
should we backup the current configuration and recover after test ?

Reply:
Good question! no need to do this here, since it is a fresh setup, no changes need to be done here.
the only point needs to be checked here is that, it should not delete any backup files which are already there.

~~~The pam-config --create command creates a simple Unix authentication configuration. Pre-existing configuration files not maintained by pam-config are overwritten, but backup copies are kept as *.pam-config-backup.~~~